package.json

```diff
@@ -3,7 +3,7 @@
   "private": true,
   "scripts": {
     "dev": "nuxt dev",
-    "build": "nuxt generate",
+    "build": "nuxt build",
```

Comment:
Let's also add a decision log entry for why we're switching to nuxt build. I've written these reasons to switch down in my notes:

-  Blog pagination is difficult with generate
- Build time
- Dynamic routes configuration is hard to maintain